default/scripting/techs/ship_weapons/fighters/fighters.py~

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Is "py~" correct extension?

Reply:
"yes", for a emacs file backup... thanks for catching